awx/ui/src/screens/Inventory/shared/Inventory.helptext.js

```diff
@@ -0,0 +1,196 @@
+/* eslint-disable react/destructuring-assignment */
+import React from 'react';
+import { t, Trans } from '@lingui/macro';
+import { Link } from 'react-router-dom';
+
+const ansibleDocUrls = {
+  ec2: 'https://docs.ansible.com/ansible/latest/collections/amazon/aws/aws_ec2_inventory.html',
+  azure_rm:
+    'https://docs.ansible.com/ansible/latest/collections/azure/azcollection/azure_rm_inventory.html',
+  controller:
+    'https://docs.ansible.com/ansible/latest/collections/awx/awx/tower_inventory.html',
+  gce: 'https://docs.ansible.com/ansible/latest/collections/google/cloud/gcp_compute_inventory.html',
+  insights:
+    'https://docs.ansible.com/ansible/latest/collections/redhatinsights/insights/insights_inventory.html',
+  openstack:
+    'https://docs.ansible.com/ansible/latest/collections/openstack/cloud/openstack_inventory.html',
+  satellite6:
+    'https://docs.ansible.com/ansible/latest/collections/theforeman/foreman/foreman_inventory.html',
+  rhv: 'https://docs.ansible.com/ansible/latest/collections/ovirt/ovirt/ovirt_inventory.html',
+  vmware:
+    'https://docs.ansible.com/ansible/latest/collections/community/vmware/vmware_vm_inventory_inventory.html',
+};
+
+const getInventoryHelpTextStrings = {
+  labels: t`Optional labels that describe this inventory,
+          such as 'dev' or 'test'. Labels can be used to group and filter
+          inventories and completed jobs.`,
+  variables: () => {
```

Comment:
Nitpick: I don't see a need for this to be a function when `jsonExample` and `yamlExample` can be hard-coded into the returned JSX